rust_src/src/windows.rs

```diff
@@ -1456,8 +1466,48 @@ pub fn scroll_up(arg: LispObject) {
 /// If ARG is the atom `-', scroll upward by nearly full screen.
 /// When calling from a program, supply as argument a number, nil, or `-'.
 #[lisp_fn(min = "0", intspec = "^P")]
-pub fn scroll_down(arg: LispObject) {
-    unsafe { scroll_command(arg, -1) };
+pub fn scroll_down(arg: Option<InteractiveNumericPrefix>) {
+    scroll_command(arg, -1);
+}
+
+// Scroll selected window up or down.  If N is nil, scroll upward by a
```

Comment:
Should be `///`?

Reply:
My understanding is the triple quote is for API level docs. Most of the internal functions we only use double slash. Did I misunderstand? Or are you suggesting this is useful enough to be raised up to the API docs?